MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [IKechukwu]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
[OPTIONAL] I thing this lines of the MIT license is not necessary. You can remove them.